bug: NPE on first launch of application.

<!-- Checklist for reporting a bug
- Check the issue tracker. The issue you have may have already been reported.
- Ensure you have tested the latest developers version.
-->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
Upon a fresh clone of the repository and build using the Android studio, I got a crash right at the right of the application.

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
It shouldn't crash.

**Steps to reproduce the behavior:**
1. Delete the application if you already have it along with it's data.
2. Build and run it again. You should get a crash.
<!--
2. Click on '....'
3. Scroll down to '....'
4. See error 
-->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem otherwise delete this heading. -->
N/A

**Environment**
- Version of Kiwix Android : Latest Master Branch
- Device : Oppo f1
- OS version : Android 5.1.1

**Logs**
<!-- If applicable, add logs to help the developers in identifying your problem otherwise delete this heading. -->
```
03-02 17:15:35.575 12863-12863/org.kiwix.kiwixmobile E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.kiwix.kiwixmobile, PID: 12863
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.kiwix.kiwixmobile/org.kiwix.kiwixmobile.splash.SplashActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.kiwix.kiwixmobile.utils.SharedPreferenceUtil.nightMode()' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2505)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2577)
        at android.app.ActivityThread.access$1000(ActivityThread.java:164)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1462)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:159)
        at android.app.ActivityThread.main(ActivityThread.java:5541)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:964)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:759)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.kiwix.kiwixmobile.utils.SharedPreferenceUtil.nightMode()' on a null object reference
        at org.kiwix.kiwixmobile.base.BaseActivity.getTheme(BaseActivity.java:50)
        at android.content.Context.isOppoStyle(Context.java:3464)
        at com.android.internal.policy.impl.PhoneWindow.<init>(PhoneWindow.java:319)
        at com.android.internal.policy.impl.Policy.makeNewWindow(Policy.java:60)
        at com.android.internal.policy.PolicyManager.makeNewWindow(PolicyManager.java:57)
        at android.app.Activity.attach(Activity.java:6049)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2439)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2577) 
        at android.app.ActivityThread.access$1000(ActivityThread.java:164) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1462) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:159) 
        at android.app.ActivityThread.main(ActivityThread.java:5541) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:372) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:964) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:759) 

```

--- a/app/src/main/java/org/kiwix/kiwixmobile/base/BaseActivity.java
+++ b/app/src/main/java/org/kiwix/kiwixmobile/base/BaseActivity.java
@@ -47,7 +47,7 @@ public abstract class BaseActivity extends AppCompatActivity {
   @Override
   public Resources.Theme getTheme() {
     Resources.Theme theme = super.getTheme();
-    if (sharedPreferenceUtil.nightMode()) {
+    if (sharedPreferenceUtil != null && sharedPreferenceUtil.nightMode()) {
       setTheme(R.style.AppTheme_Night);
     } else {
       theme.applyStyle(R.style.StatusBarTheme, true);

